web/packages/core/src/internal/player/inner.tsx

```diff
@@ -1265,7 +1278,24 @@ export class InnerPlayer {
                 );
             }
         }
-        input.value = "";
+        this.virtualKeyboard.value = "";
```

Comment:
Relates to advanced text input so probably needs to be handled in a follow-up, but I have discovered that by clearing the input value on every input, when you click a suggestion with Android GBoard, it does not fire the expected deleteContentBackward event(s) if the suggestion has different characters from the entered text, probably because it does not think it has to do so as there is no text in the text field.

Maybe we can store and clear the thing we want to type as a `data-*` attribute instead, and leave the input alone unless a new text area gets focused, or sets its value to the actual text field content from the Rust side.

Reply:
Yeah, I've been thinking about this, but that's a lot of work: you need to sync not only the text but also the caret from both sides (Rust, web). Also there's the issue of multiline fields, text restrictions, position of the text field, available glyphs, etc.